spec/dummy/config/initializers/active_admin.rb

```diff
@@ -0,0 +1,262 @@
+ActiveAdmin.setup do |config|
+  # == Site Title
+  #
+  # Set the title that is displayed on the main layout
+  # for each of the active admin pages.
+  #
+  config.site_title = 'Dummy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.